extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -86,22 +97,19 @@ export class AssessmentDetailsHeader {
 	}
 
 	// function to populate the values of properties displayed in the cards.
-	public async populateAssessmentDetailsHeader(): Promise<void> {
-
+	public async populateAssessmentDetailsHeader(migrationStateModel: MigrationStateModel): Promise<void> {
+		// this value is populated to handle the case when user selects a target type and want to resume later.
+		this._targetSelectionDropdown.value = this.getTargetTypeBasedOnModel(migrationStateModel._targetType);
 		const assessmentHeaderValues = [
-			{
-				value: this._getTargetPlatformLabel(this._migrationStateModel?._targetType)
-			},
 			{
 				// TODO(stutijain): replace below value with recommended config
 				value: "10 available"
```

Comment:
This value is not available as of now, if that there please remove hardcoded value.

Reply:
This is dependent on Ritik's changes. It will be passed from sku recommendation page as parameter. I have added this as TODO remaining in the description.

Reply:
What exact is this change on SKU Recommendation page?

Reply:
We show Recommended config on the cards on sku recommendation page. Same is to be shown here. We need to calculate that recommended config based on existing data. Probably you will have a function to get that config, We can keep it in utils and use the same function on both the pages.